src/components/Timeline/TimelineItem.tsx

```diff
@@ -2,11 +2,13 @@ interface TimelineItemProps {
   title: string;
   time: string;
   text: string;
+  color: string | undefined;
 }
 
-const TimelineItem: React.FC<TimelineItemProps> = ({ title, text, time, children }) => {
+const TimelineItem: React.FC<TimelineItemProps> = ({ title, text, time, color, children }) => {
   return (
     <li className="timeline-item">
+      <div className="timelile-point" style={{ backgroundColor: color }}></div>
```

Comment:
- Remove style prop
Implement this requirement by creating a className for your <TimelineItem /> with modifiers of statuses.